velox/docs/develop/scalar-functions.rst

```diff
@@ -870,12 +870,14 @@ Use exec::registerVectorFunction to register a stateless vector function.
         const std::string& name,
         std::vector<FunctionSignaturePtr> signatures,
         std::unique_ptr<VectorFunction> func,
+        VectorFunctionMetadata metadata = {},
         bool overwrite = true)
 
 exec::registerVectorFunction takes a name, a list of supported signatures
-and unique_ptr to an instance of the function. An optional “overwrite” flag
-specifies whether to overwrite a function if a function with the specified
-name already exists.
+and unique_ptr to an instance of the function. It is optional to provide
```

Comment:
> metadata flag

'metadata' is a struct with multiple flags, hence, it is incorrect to refer to it as a 'flag'.

Consider,

```
It takes an optional 'metadata' parameter that specifies whether a function is deterministic, has default null behavior, and other properties. A helper VectorFunctionMetadataBuilder class allows to easily construct 'metadata'. For example,...
```